### PR TITLE
Update drawbot to 3.94

### PR DESCRIPTION
**Pull request by gobinathm**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/drawbot.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 cask 'drawbot' do
   version '3.89'
-  sha256 '9792ea095e3b900cdd22d0fa7b1c5c2bed5a450e42ec1fdf4f10e0b3af616e2e'
+  sha256 '6c8e43816ea05efa53439d2f2e147443d4e45aec06d7fc5676c05d49d17a2764'
 
   # typemytype.com/drawBot was verified as official when first introduced to the cask
   url 'http://static.typemytype.com/drawBot/DrawBot.dmg'
   appcast 'https://raw.githubusercontent.com/typemytype/drawbot/master/drawBotSettings.py',
```
Comment (victorpopkov) [inline Casks/drawbot.rb]: The appcast location has changed:
```ruby
appcast  'https://raw.githubusercontent.com/typemytype/drawbot/master/drawBot/drawBotSettings.py',
         checkpoint: 'b2f58ba11dcf9e3fef254e84aed69e185e72b18236ef63be6cdf86a86ae8bb53'
```

File: Casks/drawbot.rb [hunk 2]
```diff
@@ -1,11 +1,11 @@
 cask 'drawbot' do
   version '3.89'
```
Comment (victorpopkov) [inline Casks/drawbot.rb]: The latest version according to latest appcast is `3.94`.

**Review by victorpopkov — CHANGES_REQUESTED** [review]

File: Casks/drawbot.rb (hunk 2 above)
Comment (gobinathm) [inline Casks/drawbot.rb]: @victorpopkov This has been updated.

File: Casks/drawbot.rb (hunk 1 above)
Comment (gobinathm) [inline Casks/drawbot.rb]: @victorpopkov this has been corrected

**Review by victorpopkov — APPROVED** [review]

**Pull request merged**